client/src/pages/EditProfile.js

```diff
@@ -52,13 +52,20 @@ export default function EditProfile() {
     const userData = user.profile;
 
     const [cuisineChipData, setCuisineChipData] = React.useState(
-        userData.favoriteCuisine.map((cuisine, index) => ({
-            key: index,
-            label: cuisine,
-        }))
+        !userData.favoriteCuisine
+            ? ""
```

Comment:
If we set this to [] instead of "", then we don't have to check a second time below if cuisineChipData exists